stdlib/@tests/stubtest_allowlists/common.txt

```diff
@@ -4,6 +4,7 @@
 
 # Please keep sorted alphabetically
 
+_asyncio.Future.__init__  # Usually initialized from c object
```

Comment:
duplicate of existing allowlist entry for `asyncio.Future.__init__` and `asyncio.futures.Future.__init__`